### PR TITLE
Add smirnoff

### PR DESCRIPTION
**Pull request by sjothunj**
Please review this, @casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/light-fire/stop-drop-roll/stop-drop-roll.md [hunk 1]
```diff
@@ -6,3 +6,5 @@ Do you?
 [Turn on the chocolate sprinkler!](../sprinkler/eating-chocolate.md)
 
 [Scream for help?](../../sleep/scream-help/scream-help.md)
+
+[Grab your bottle of Marshmallow flavored vodka?] (/smirnoff/vodka.md)
```
Comment (casey-collab) [inline english/light-fire/stop-drop-roll/stop-drop-roll.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/light-fire/stop-drop-roll/smirnoff/smirnoff.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+You desperately take a huge swig from the bottle.
```
Comment (casey-collab) [inline english/light-fire/stop-drop-roll/smirnoff/smirnoff.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/light-fire/stop-drop-roll/stop-drop-roll.md (hunk 1 above)
Comment (cbuckey-uda) [inline english/light-fire/stop-drop-roll/stop-drop-roll.md]: It probably shouldn't have the slash at the beginning.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**